scylla-cdc/src/test_utilities.rs

```diff
@@ -62,11 +67,23 @@ fn get_uri() -> String {
     std::env::var("SCYLLA_URI").unwrap_or_else(|_| "127.0.0.1:9042".to_string())
 }
 
-pub async fn prepare_db() -> anyhow::Result<(Arc<Session>, String)> {
+pub async fn prepare_db(
+    schema: &[String],
+    replication_factor: u8,
+) -> anyhow::Result<(Arc<Session>, String)> {
     let uri = get_uri();
     let session = SessionBuilder::new().known_node(uri).build().await.unwrap();
     let shared_session = Arc::new(session);
 
-    let ks = create_test_db(&shared_session).await?;
+    let ks = create_test_db(&shared_session, schema, replication_factor).await?;
     Ok((shared_session, ks))
-}
\ No newline at end of file
+}
+
+pub async fn prepare_simple_db() -> anyhow::Result<(Arc<Session>, String)> {
```

Comment:
I believe the commit diffs will look a bit nicer if you introduce and start using this function in the previous commit. You won't have to do so many `db` -> `simple_db` renames.

Besides, it looks like `prepare_simple_db`'s body could be simplified to just:

```rust
{
    prepare_db(&[get_create_table_query()], 1).await
}
```